src/cutapplier.cc

```diff
@@ -21,10 +21,10 @@ using std::cout;
 using std::endl;
 
 int main(int argc, char *argv[]) {
-	if(argc != 5 ){
```

Comment:
Generally it's nice to keep the last argument as the output filename: I'd suggest for the friendtree making this the 2nd argument, ie: immediately after the input file, rather than the last argument.